x/axelarnet/migrations/v014/store.go

```diff
@@ -0,0 +1,49 @@
+package v014
+
+import (
+	"github.com/axelarnetwork/axelar-core/utils"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// MigrateStore performs in-place store migrations from v0.13 to v0.14. The
+//// migration includes:
+// - remove all path
+// - remove all chain <-> asset mapping
+func MigrateStore(ctx sdk.Context, storeKey sdk.StoreKey, cdc codec.BinaryCodec) error {
+	store := getStore(ctx.KVStore(storeKey), cdc)
+	deleteAllPath(store)
+	deleteAllChainByAsset(store)
+	deleteAllAssetByChain(store)
+
+	return nil
+}
+
+func deleteAllPath(store utils.KVStore) {
+	iter := store.Iterator(pathPrefix)
+
+	for ; iter.Valid(); iter.Next() {
+		store.Delete(iter.GetKey())
+	}
+
+}
+
+func deleteAllChainByAsset(store utils.KVStore) {
+	iter := store.Iterator(chainByAssetPrefix)
+
+	for ; iter.Valid(); iter.Next() {
+		store.Delete(iter.GetKey())
+	}
+}
+
+func deleteAllAssetByChain(store utils.KVStore) {
+	iter := store.Iterator(assetByChainPrefix)
+
+	for ; iter.Valid(); iter.Next() {
+		store.Delete(iter.GetKey())
+	}
+}
```

Comment:
the only thing that changes across these three functions is the prefix. Why not have a single function that iterates over whatever prefix you pass to it?

Reply:
sounds good